### DISCUSSION TITLE
[Q] Difficulty doing push in branch

### DISCUSSION THREAD

**Discussion by alanrps** [exceljs/exceljs#2380] [Q&A]
## 💬 Error with permission when uploading fixes

![image](https://user-images.githubusercontent.com/33161962/114326928-6cf7ee80-9b0d-11eb-8492-9eff0f1e81b3.png)

I can't go up changing to pull request due to lack of permission, I need help.

**Comment by skypesky** [comment]
@alanrps Maybe this will work for you!
https://opensource.com/article/19/7/create-pull-request-github
